docs/product-user-guide/src/main/asciidoc/drl-rules-central-create-proc.adoc

```diff
@@ -5,14 +5,14 @@ You can create and manage DRL files for your project in {CENTRAL}. In each DRL f
 
 .Procedure
 . Go to *Menu* -> *Design* -> *Projects* and click the project name.
-. Click *Create New Asset* -> *DRL file*.
+. Click *Add Asset* -> *DRL file*.
 . Enter an informative *DRL file* name and select the appropriate *Package*. The package that you specify must be the same package where the required data objects have been assigned or will be assigned.
 +
-You can also select *Use Domain Specific Language (DSL)* if any DSL assets have been defined in your project (in the *Domain Specific Language Definitions* panel in the *Project Explorer*). These DSL assets will then become usable objects for conditions and actions that you define in the DRL designer.
+You can also select *Show declared DSL sentences* if any Domain Specific Language (DSL) assets have been defined in your project. These DSL assets will then become usable objects for conditions and actions that you define in the DRL designer.
```

Comment:
Domain Specific Language. I would make this LC.

Reply:
Good call. This was a direct UI reference previously and I updated it (as you see) and forgot to lc.